### PR TITLE
updated list of gluonnlp tutorials

### PR DESCRIPTION
**Pull request by IvyBazan**
Added GluonNLP tutorials to the Gluon text tutorials.
A preview can be found here: http://35.175.223.197/guide/packages/gluon/text/index.html

### CODE REVIEW

**Comment by IvyBazan** [comment]
@aaronmarkham

File: python/guide/packages/gluon/text/index.rst [hunk 1]
```diff
@@ -17,6 +17,96 @@ These tutorials will help you learn how to create and use models that work with
...
+      :link: gnmt.html
+             
+      How to create a model that translates between English and Vietnamese.
+
+   .. card::
+      :title: Transformer Models with GluonNLP
+      :link: transformer.html
+    
+      How to use a pretrained Transformer model and how to train one that translates between English and German.
+Text Tutorials
```
Comment (aaronmarkham) [inline python/guide/packages/gluon/text/index.rst]: I'd put a line break here for readability and also Sphinx really likes line breaks.

File: python/guide/packages/gluon/text/index.rst [hunk 2]
```diff
@@ -17,6 +17,96 @@ These tutorials will help you learn how to create and use models that work with
...
```
Comment (aaronmarkham) [inline python/guide/packages/gluon/text/index.rst]: I think this should be removed.

Comment (IvyBazan) [reply python/guide/packages/gluon/text/index.rst]: Updated with correct content and added line break

**Pull request merged**